docs/PYMEImage_rendering.rst

```diff
@@ -0,0 +1,33 @@
+.. _pymeimage_rendering:
+
+
+Rendering Images and Movies with PYMEImage
+===========================================
+
+PYMEImage facilities exporting image data, and rendering, two similar but distinct tasks.
+Exporting images via `File > Save As` and `File > Export Cropped` will save the image data to disk,
+while here we will cover exporting and saving rendered views of these data.

```

Comment:
See comments above - I don't really like the rendering term - we need to be a little more explicit that this is exporting images with LUT etc ... applied. It's probably also worth putting a sentence in about when you would (for presentations etc ) and wouldn't (for analysis) use data exported in this way.